pkg/bootstrap/option/instances.go

```diff
@@ -247,3 +255,7 @@ func ProvCert(value string) Instance {
 func DiscoveryHost(value string) Instance {
 	return newOption("discovery_host", value)
 }
+
+func DualStack(value bool) Instance {
```

Comment:
No need for this function. It was used for "envoy.reloadable_features.listener_wildcard_match_ip_family": {{ .dual_stack }}, flag. But those are not needed.

Reply:
okay, I will remove it

Reply:
done